tests/crypto/enclave/enc/CMakeLists.txt

```diff
@@ -26,21 +26,51 @@ set(SRCS
     ../../utils.c)
 
 if (OE_SGX)
-  list(APPEND SRCS ../../../../common/sgx/rand.S ../../cpu_entropy_test.c)
+  list(APPEND SRCS ${PROJECT_SOURCE_DIR}/common/sgx/rand.S
+       ../../cpu_entropy_test.c)
 endif ()
 
 add_enclave(
   TARGET
-  crypto_enc
+  crypto_mbedtls_enc
   UUID
   f0be7db0-ce7c-4dc4-b8c8-b161f4216225
+  CRYPTO_LIB
+  mbedtls
   SOURCES
-  ${SRCS}
-  crypto_t.c)
+  enc.c
+  crypto_t.c
+  ${SRCS})
+
+#enclave_link_libraries(crypto_mbedtls_enc crypto_object)
```

Comment:
Remove unused directives here and on line 67

Reply:
Done